@kiva/kv-components/src/vue/KvCartPill.vue

```diff
@@ -29,15 +29,33 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		milestonesNumber: {
+			type: Number,
+			default: 1,
+		},
+		isCloseNextMilestone: {
+			type: Boolean,
+			default: false,
+		},
 	},
 	setup(props) {
 		const {
 			borrowerName,
+			milestonesNumber,
+			isCloseNextMilestone,
 		} = toRefs(props);
 
 		const pillCopy = computed(() => {
+			if (isCloseNextMilestone.value) {
```

Comment:
Could we just check `milestonesNumber.value === 1`?

Reply:
I think is a different use case as we also have this below:
<img width="363" alt="Screenshot 2025-03-11 at 3 05 16 p m" src="https://github.com/user-attachments/assets/df1d4a51-9c23-4b23-aaa6-6ddadd696947" />

Reply:
Oh, you're right, it's when you're one loan away from the next milestone.